robotoff/prediction/ocr/category_from_AOC.py

```diff
@@ -0,0 +1,109 @@
+import re
+from typing import List, Optional, Union
+
+from robotoff.off import normalizing
+from robotoff.prediction.types import Prediction, PredictionType
+from robotoff.taxonomy import get_taxonomy
+from robotoff.utils import get_logger
+
+from .dataclass import OCRField, OCRRegex, OCRResult, get_text
+
+logger = get_logger(__name__)
+
+
+def category_taxonomisation(lang, match) -> Optional[str]:
+    """Function to match categories detected via AOP REGEX with categories
+    taxonomy database. If no match is possible, we return None."""
```

Comment:
style: normally, in docstring, the last triple quotes is on a new line, this is a convention
```suggestion
def category_taxonomisation(lang, match) -> Optional[str]:
    """Function to match categories detected via AOP REGEX with categories
    taxonomy database. If no match is possible, we return None.
    """
```